Dockerfile

```diff
@@ -1,18 +1,22 @@
 FROM quay.io/rbreslow/steamcmd:slim
 
-MAINTAINER Rocky Breslow <breslowrocky@gmail.com>
+LABEL maintainer="Rocky Breslow <breslowrocky@gmail.com>"
+
+USER root
```

Comment:
We have to switch to the `root` user first to have permissions to manage apt.

Reply:
yes